packages/devtools_testing/pubspec.yaml

```diff
@@ -9,7 +9,7 @@ environment:
 
 dependencies:
   devtools_app: ^0.1.15
```

Comment:
Should this be more constrained too?  Can't really imagine how that would mess anything up but would mean be consistent in all pubspec's.  Maybe a script that updates all the pubspecs, version.dart, etc.  So nothing is missed?

Reply:
goot catch. Fixed this constraint. Also added a shell script to update the versions as well as devtools.dart
Updated tool/README.md to use the new script.
@kenzieschmoll also noticed that we were missing steps to update some of the shared packages in tool./README.md. Please verify that this order is the order you used last time you published.

Reply:
The order I published in is:
devtools_shared
devtools_server
devtools_testing
devtools_app
devtools

Reply:
updated to match the order